### ISSUE TITLE
Unhandled exception when remove -> local

### ISSUE THREAD

**Issue by alump** [vaadin/px-data-grid#60]
This unhandled exception happens if move from remote dataprovider back to local data. Could this exception be the reason why this move does not work correctly?

```
vaadin-grid-a11y-mixin.html:103 Uncaught TypeError: Cannot read property 'parentNode' of null
    at Array.from.forEach.sorter (vaadin-grid-a11y-mixin.html:103)
    at Array.forEach (<anonymous>)
    at HTMLElement._a11yUpdateSorters (vaadin-grid-a11y-mixin.html:97)
    at HTMLElement._onSorterChanged (vaadin-grid-sort-mixin.html:93)
    at HTMLElement._pathOrDirectionChanged (vaadin-grid-sorter.html:192)
    at Object.runMethodEffect [as fn] (property-effects.html:797)
    at runEffectsForProperty (property-effects.html:160)
    at runEffects (property-effects.html:126)
    at HTMLElement._propertiesChanged (property-effects.html:1648)
    at HTMLElement._flushProperties (property-accessors.html:549)
```

**Comment by alump** [comment]
Is the reason here that we maybe try to still use inmemory sorters, when we should replace sorting to be handled by remote data provider API?

**Comment by limonte** [comment]
Depends on https://github.com/vaadin/vaadin-grid/issues/1126

**Comment by alump** [comment]
Moved to later delivery as this needs fix to vaadin-grid. Or then we need to fix it with workaround.

**Comment by alump** [comment]
I think I can close this, as we do not use items anymore.
